src/main/kotlin/be/zvz/koreanbots/KoreanBots.kt

```diff
@@ -1,83 +1,70 @@
-        lateinit var botList: BotList
+    fun updateBotServers(id: String, servers: Int) {
+        val (_, _, rst) = Fuel.post("/bots/$id/stats")
+            .header(Headers.AUTHORIZATION, token)
+            .objectBody(ServersUpdate(servers), mapper = mapper)
+            .responseObject<ResponseWrapper<Unit>>(mapper = mapper)
 
-        "/bots/get"
-                .httpGet(listOf(Pair("page", page)))
-                .responseObject<BotList> { _, _, result ->
-                    botList = result.get()
+        handleResponse(rst)
+    }
 
-                    botList.message?.let {
-                        throw RequestException(botList.code, it)
-                    }
-                }
+    fun getUserInfo(id: String): User {
+        val (_, _, rst) = Fuel.get("/users/$id")
+            .responseObject<ResponseWrapper<User>>(mapper = mapper)
 
-        return botList
+        return handleResponse(rst)
+            ?: throw AssertionError("Request Success, but Data Doesn't Exist") // This may not occur, but in case of server api error
     }
 
-    fun updateServerCount(serverCount: Int): ServersResponse {
-        lateinit var servers: ServersResponse
+    private fun <T> handleResponse(result: Result<ResponseWrapper<T>, FuelError>): T? {
+        val wrapper = result.getOrElse { throw RequestFailedException(it.message.orEmpty()) }
 
-        "/bots/servers"
-                .httpPost()
-                .header(Pair("token", token))
-                .objectBody(ServersRequest(serverCount))
-                .responseObject<ServersResponse> { _, _, result ->
-                    servers = result.get()
+        if (wrapper.code !in 200..299)
+            throw RequestFailedException("API responded ${wrapper.code}: ${wrapper.message}")
 
-                    servers.message?.let {
-                        throw RequestException(servers.code, it)
-                    }
-                }
-
-        return servers
+        return wrapper.data
     }
 
-    fun checkVoted(userId: Int): Voted {
-        lateinit var voted: Voted
-
-        "/bots/voted/${userId}"
-                .httpGet()
-                .responseObject<Voted> { _, _, result ->
-                    voted = result.get()
+    companion object {
+        const val BASE_URL = "https://koreanbots.dev/api/v2"
 
-                    voted.message?.let {
-                        throw RequestException(voted.code, it)
-                    }
-                }
+        private val mapper = ObjectMapper()
+            .registerKotlinModule()
+            .enable(DeserializationFeature.READ_ENUMS_USING_TO_STRING)
+            .enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING)
 
-        return voted
+        init {
+            FuelManager.instance.basePath = BASE_URL
+            FuelManager.instance.baseHeaders = mapOf(
+                Headers.USER_AGENT to "Kotlin SDK(alpha, https://github.com/nkgcp/Koreanbots-Kotlin-SDK)"
+            )
+        }
```

Comment:
- `companion object` 외부로 이동
- User-Agent 변경 제거

```suggestion
        init {
            FuelManager.instance.basePath = BASE_URL
        }
```

Reply:
User-Agent가 비어있을 경우 403 Forbidden이 반환되는 이슈가 있는 것으로 보입니다.
User-Agent가 있어도 문제가 없다고 판단되는 바 삭제는 하지 않았으며, 
제거해야하는 이유가 있으신 경우 다시 말씀해주시기 바랍니다.
User-Agent는 디스코드 API의 User-Agent를 참고하였으며, 
깃허브 레포 url은 상위 레포지토리로, 버전은 동기화되도록 수정했습니다.

Reply:
`static init`을 사용한 이유는 KoreanBots 인스턴트를 여러 개 생성하는 예외적인 상황을 생각한 것인데
`FuelManager`를 사용하게 변경하는 것이 더 좋은 것 같습니다.

Reply:
User-Agent 변경하면서 `FuelManager` 인스턴트로 같이 변경하는 것이 좋아보입니다.
머지 전에 반영하겠습니다.